services/storage/src/simcore_service_storage/constants.py

```diff
@@ -41,3 +41,6 @@
 
 
 EXPAND_DIR_MAX_ITEM_COUNT: Final[int] = 1000
+
+
+EXPORTS_S3_PREFIX: Final[str] = "exports"
```

Comment:
I have seen `exports` in multiple files. should this prefix not be used more?

Reply:
It's already used all over the place